Project-Aurora/Project-Aurora/ConfigUI.xaml.cs

```diff
@@ -107,7 +107,7 @@ public bool ShowHidden
 
     internal void DisplayIfNotSilent()
     {
-        if (App.IsSilent)
+        if (App.IsSilent || Global.Configuration.StartSilently)
```

Comment:
This will block aurora from showing it's window while opening from exe

Reply:
I can revert it - I thought the user chat in Discord meant this behavior should be there (since it was before).